frontend/src/metabase/core/components/FormField/FormField.tsx

```diff
@@ -49,6 +52,7 @@ const FormField = forwardRef(function FormField(
             {title && (
               <FieldLabel hasError={hasError} htmlFor={htmlFor}>
                 {title}
+                {!!optional && !hasError && t` (optional)`}
```

Comment:
Would `findByLabelText` work for an optional field by its label (without `(optional)` part)? I believe not since we're adding it as inline text. Could you add a unit test for that, and if it doesn't work, wrap it in a `span` or something?

Reply:
you're right. I had to move it outside the label. added a test.